feature_tests/js/api.mjs

```diff
@@ -105,7 +105,7 @@ export class Opaque {
     const diplomat_MyStruct_extracted_d = s["d"];
     const diplomat_MyStruct_extracted_e = s["e"];
     const diplomat_MyStruct_extracted_f = s["f"];
-    if (!diplomat_MyStruct_extracted_f.length || !diplomat_MyStruct_extracted_f.codePointAt || diplomat_MyStruct_extracted_f.length != 1) { throw new TypeError("Expected single-character string for `char` parameter diplomat_MyStruct_extracted_f, found " + diplomat_MyStruct_extracted_f); }
+    if (!diplomat_MyStruct_extracted_f.length || !diplomat_MyStruct_extracted_f.codePointAt || [...diplomat_MyStruct_extracted_f].length != 1) { throw new TypeError("Expected single-character string for `char` parameter diplomat_MyStruct_extracted_f, found " + diplomat_MyStruct_extracted_f); }
     const diplomat_out = wasm.Opaque_assert_struct(this.underlying, diplomat_MyStruct_extracted_a, diplomat_MyStruct_extracted_b, diplomat_MyStruct_extracted_c, diplomat_MyStruct_extracted_d, diplomat_MyStruct_extracted_e, diplomat_MyStruct_extracted_f.codePointAt(0));
```

Comment:
Nit (optional): This is JavaScript.  I'm not a fan of this brand checking.  How about using optional chaining and then checking for undefined, like

```javascript
const cp = diplomat_MyStruct_extracted_f?.codePointAt(0);
if (!cp && cp !== 0) {
    // throw TypeError
}
// use cp
```